raw.go

```diff
@@ -8,9 +8,13 @@ import (
 
 // RawSpan encapsulates all state associated with a (finished) Span.
 type RawSpan struct {
-	// The RawSpan embeds its Context. Those recording the RawSpan
-	// should also record the contents of its Context.
-	Context
+	// The RawSpan embeds its SpanMetadata. Those recording the RawSpan
+	// should also record the contents of its SpanMetadata.
+	SpanMetadata
+
+	// The SpanID of this SpanMetadata's first intra-trace reference (i.e.,
+	// "parent"), or 0 if there is no parent.
+	ParentSpanID uint64
```

Comment:
despite the multiple-parent semantics of OT proper, I decided it better to stick with the simple ParentSpanID model for `basictracer` per the `basic` prefix.